test/standalone/Soil/climate_drivers.jl

```diff
@@ -150,7 +152,8 @@ include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"))
         @test ClimaLSM.surface_emissivity(model, Y, p) == emissivity
         @test ClimaLSM.surface_evaporative_scaling(model, Y, p) == FT(1)
         @test ClimaLSM.surface_height(model, Y, p) == z_sfc
-        @test ClimaLSM.surface_albedo(model, Y, p) == albedo
+        @test ClimaLSM.surface_albedo(model, Y, p) ==
+              FT((0.5 * PAR_albedo) + (0.5 * NIR_albedo))
```

Comment:
small note. `a*0.5` will always convert to Float64, but if you write `a/2` it will preserve the type of `a`.